ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/PauseQueryEntity.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PauseQueryEntity extends KsqlEntity {
```

Comment:
Is this used anywhere? I don't see this actually returned from anything. Same for ResumeQueryEntity below.

Reply:
Ah!  It is used here: https://github.com/confluentinc/ksql/pull/9203/files#diff-fea2df4b7059566421cb038afd9ded3c367b19c5d42f59b87284c75f12160b5bR198-R201

This class helps with the formatting for things in the Console.

Reply:
But PauseQueryEntity is never actually returned from anywhere, is it? When I looked earlier it seemed like all the infrastructure for pretty-printing was dead code since it's never called. Very possible I missed something, though.

Reply:
Ok....  so I was cribbing off of `TERMINATE` when I implemented this.

It looks like there a separate `Executor` for Terminate commands which returns the `TerminateQueryEntity`.  

Likely I need to find the right `Executor` and add returns for the Pause and Resume query entities?  Does that seem reasonable?

Reply:
I think we can actually just remove all this. It looks like the only reason there's a separate `TerminateQueryExecutor` (from which `TerminateQueryEntity` is returned) at all is to handle pull query termination. The regular persistent query terminations go through the same distributing executor as your new pause/resume statements, and there's a different KsqlEntity already being returned from there. The CLI should know how to pretty-print those already.

You can run a quick manual check to see if the CLI output from pause/resume looks acceptable with the PR as is. If so, go ahead and remove all this as dead code.